src/uct/base/uct_iface.h

```diff
@@ -153,6 +153,42 @@ enum {
                                        UCT_EP_PARAM_FIELD_IFACE_ADDR), \
                     "UCT_EP_PARAM_FIELD_DEV_ADDR and UCT_EP_PARAM_FIELD_IFACE_ADDR are not defined")
 
+/**
+ * Returns 0 if @a _params field mask does not have the required flags set.
+ */
+#define UCT_EP_PARAMS_CHECK_ADDRS(_params, _flags, _err_message, ...) \
```

Comment:
The body of this macro doesn't have to do anything with "addrs", but only flags. Maybe rename accordingly?